decentraland/unity-explorer
# fix: copy offline realm adapter value

## What does this PR change?

Copies the value of the realm adapter value so it can be consistent with the `offline:offline` adapter

## How to test the changes?

1. Download the build from this PR and open it entering directly into the following different realms (WinOS: `"C:\Users\[YOUR-USER]\Downloads\Decentraland_windows64\Decentraland.exe" --realm TARGET-REALM-HERE -- --debug` / MacOS: `open Decentraland.app --args --realm TARGET-REALM-HERE --debug`) and confirm that you can enter the world (and are not "trapped" returning constantly to the auth screen):
* `onboardingdcl.dcl.eth`
* `https://peer-testing.decentraland.org`
* `https://peer-ec1.decentraland.org`
* `https://peer-wc1.decentraland.org`
* `https://peer-eu1.decentraland.org`
* `https://interconnected.online`

2. Additionally open the PR build normally without connecting to any specific realm and confirm that it still loads the Genesis City correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
